fhttp/http_client.go

```diff
@@ -409,8 +419,10 @@ func NewStdClient(o *HTTPOptions) (*Client, error) {
 		}
 	}
 	client := Client{
-		url: o.URL,
-		req: req,
+		url:          o.URL,
+		path:         req.URL.Path,
```

Comment:
I suspect it's not working because you change the path and not the full url

see comment about test

Reply:
Correct. I will fix that. I did not reproduce this problem because our use case only needed path to be replaced.